### DISCUSSION TITLE
Millisecond Timer would be useful

### DISCUSSION THREAD

**Discussion by smp-marco** [smplayer-dev/smplayer#570] [Ideas]
**Millisecond Timer would be useful**
When taking notes of frame times, all that we have are to the second times.
Yet when editing videos, the times displayed go to milliseconds.

Certainly for me; having the option to choose a time display with milliseconds would be very useful.
Perhaps this could be considered for future updates.

Thanks for making a great media player.
:)

**Comment by smplayer-dev** [comment]
Options -> status bar -> Show current time in milliseconds.

**Reply by smp-marco** [reply]
I my goodness; sorry Ricardo.
That's two fails.
I was only scouring preferences.
Yeah; I should have been in bed :D
